src/client/camera.h

```diff
@@ -176,10 +190,12 @@ class Camera
 	WieldMeshSceneNode *m_wieldnode = nullptr;
 
 	// draw control
-	MapDrawControl& m_draw_control;
+	MapDrawControl &m_draw_control;
 
 	Client *m_client;
 
+	CameraModeRestrictions *m_camera_mode_restrictions;
```

Comment:
this attribute must be private, we already have a setter and getter

Reply:
It is private, as far as I can tell :)